test/generaloption.py

```diff
@@ -764,6 +773,106 @@ def raise_error(msg, *args):
                               go_args=['--level-level'], envvar_prefix='GENERALOPTIONTEST', error_env_options=True,
                               error_env_option_method=raise_error)
 
+    def _match_testoption1_sysexit(self, args, msg):
+        self.mock_stderr(True)  # reset
+        self.assertErrorRegex(SystemExit, '.*', TestOption1, go_args=args)
+        stderr = self.get_stderr()
+        self.assertTrue(msg in stderr)
```

Comment:
While you're mocking stderr, you won't get any output on stderr anymore.

The tearDown will disable the mocking, sure, but it's cleaner to do that in here (since you may be calling this method several times in a single test).